cli/tests/pcluster/schemas/test_imagebuilder_schema/test_imagebuilder_schema/imagebuilder_schema_simple.yaml

```diff
@@ -0,0 +1,6 @@
+Image:
```

Comment:
Is this a test with only the required parameters? If so let's rename the test to imagebuilder_schema_required.yaml